docs/program/apis/_index.md

```diff
@@ -17,178 +17,25 @@ Different models of resources implement the same API, which [Viam SDKs expose](/
-
-```python
-my_arm_name = my_arm.get_resource_name("my_arm")
-```
-
-For more information, see the [Python SDK Docs](https://python.viam.dev/autoapi/viam/resource/base/index.html).
-
-{{% /tab %}}
-{{% tab name="Go" %}}
-
-Get the [`Name`](https://pkg.go.dev/go.viam.com/rdk/resource#Name) of the resource.
-
-**Parameters:**
-
-- None
-
-**Returns:**
-
-- `name` [(Name)](https://pkg.go.dev/go.viam.com/rdk/resource#Name): The [`Name`](https://pkg.go.dev/go.viam.com/rdk/resource#Name) of the resource, including fields for the `API` with `Type` and `SubtypeName`, and string `Remote` and `Name`.
-
-```go
-MyArmName := MyArm.Name()
-```
-
-For more information, see the [Go SDK Docs](https://pkg.go.dev/go.viam.com/rdk/resource#Resource).
-
-{{% /tab %}}
-{{< /tabs >}}
-
-### DoCommand
-
-DoCommand sends commands containing arbitrary data to the resource.
-
-{{< tabs >}}
-{{% tab name="Python" %}}
-
-**Parameters:**
-
-- `command` [(`Mapping[str, viam.utils.ValueTypes]`)](https://python.viam.dev/autoapi/viam/utils/index.html#viam.utils.ValueTypes): The command to execute.
-- `timeout` [(`Optional[float]`)](https://docs.python.org/library/typing.html#typing.Optional): An option to set how long to wait (in seconds) before calling a time-out and closing the underlying RPC call.
-
-**Returns:**
-
-- [(`Mapping[str, viam.utils.ValueTypes]`)](https://python.viam.dev/autoapi/viam/utils/index.html#viam.utils.ValueTypes): The result of the executed command.
-
-For more information, see the [Python SDK Docs](https://python.viam.dev/autoapi/viam/resource/base/index.html).
-
-{{% /tab %}}
-{{% tab name="Go" %}}
-
-**Parameters:**
-
-- `ctx` [(Context)](https://pkg.go.dev/context): A Context carries a deadline, a cancellation signal, and other values across API boundaries.
-- `cmd` [(map[string]interface{})](https://go.dev/blog/maps): The command to execute.
-
-**Returns:**
+## Robot Management APIs
 
-- [(map[string]interface{})](https://go.dev/blog/maps): The result of the executed command.
-- [(error)](https://pkg.go.dev/builtin#error): An error, if one occurred.
+### Robot API
 
-For more information, see the [Go SDK Docs](https://pkg.go.dev/go.viam.com/rdk/resource#Resource).
+All robots support the following methods through the [robot API](/program/apis/robot/):
 
-{{% /tab %}}
-{{% tab name="TypeScript" %}}
+{{< readfile "/static/include/services/apis/robot.md" >}}
 
-**Parameters:**
+### Cloud API
 
-- `command` [(StructType)](https://ts.viam.dev/types/StructType.html): The command to execute.
+The [cloud API](/program/apis/cloud/) supports the following methods (among [others](https://python.viam.dev/autoapi/viam/app/app_client/index.html#viam.app.app_client.AppClient)):
```

Comment:
I think with Jessamy's latest changes this now includes all of them - so we can also change that while we're here
```suggestion
The [cloud API](/program/apis/cloud/) supports the following methods:
```